openquake/hazardlib/tests/gsim/tusa_langer_2015_test.py

```diff
@@ -0,0 +1,112 @@
+# The Hazard Library
+# Copyright (C) 2015 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#
+# Test tables elaboratated from data provided directly from the authors.
+#
+
+"""
+Implements the tests for the set of GMPE classes included within the
+GMPE of Tusa and Langer (2015)
+"""
+from openquake.hazardlib.gsim.tusa_langer_2015 import (TusaLanger2015RepiBA08SE,
+						       TusaLanger2015RepiBA08DE,
+						       TusaLanger2015RepiSP87SE,
+						       TusaLanger2015RepiSP87DE,
+						       TusaLanger2015Rhypo)
+
+
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+
+# Discrepency percentages to be applied to all tests
+MEAN_DISCREP = 0.1
+STDDEV_DISCREP = 0.1
+
+
+class TusaLanger2015RepiBA08SEtestcase(BaseGSIMTestCase):
```

Comment:
Please rename TusaLanger2015RepiBA08SEtestcase -> TusaLanger2015RepiBA08SETestCase; idem for the other TestCase below